README.md

```diff
@@ -52,10 +57,39 @@ const pool = new Pool({
   database: 'db-name',
   max: 5
 });
-const result = await pool.query('SELECT NOW()');
+const {rows} = await pool.query('SELECT NOW()');
+console.table(rows); // prints returned time value from server
+
+await pool.end();
+connector.close();
+```
+
+### Using with MySQL
+
+Here is how to start a new
+[`mysql2`](https://www.npmjs.com/package/mysql2) connection pool.
+
+```js
+import mysql from 'mysql2/promise';
+import {Connector} from '@google-cloud/cloud-sql-connector';
+
+const connector = new Connector();
+const clientOpts = await connector.getOptions({
+  instanceConnectionName: 'my-project:region:my-instance',
+  type: 'PUBLIC',
+});
+const pool = await mysql.createPool({
+  ...clientOpts,
+  user: 'my-user',
+  password: 'my-password',
+  database: 'db-name',
+});
+const conn = await pool.getConnection();
+const [[result]] = await conn.query( `SELECT VERSION();`);
```

Comment:
honestly I just went with the first simple method I found for each of the dbs, I guess I'd prefer standardizing on `select version()` in case that method is available on PostgreSQL? 🤔

Reply:
`SELECT NOW()` is available in both mysql and postgres I know that for sure.

Reply:
oh good to know, I'll move the example to use `SELECT NOW()` then